src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -907,19 +921,23 @@ Transactor::operator()()
         std::vector<uint256> removedOffers;
         std::vector<uint256> removedTrustLines;
         std::vector<uint256> expiredNFTokenOffers;
+        std::vector<uint256> expiredCredentials;
 
         bool const doOffers =
             ((result == tecOVERSIZE) || (result == tecKILLED));
         bool const doLines = (result == tecINCOMPLETE);
         bool const doNFTokenOffers = (result == tecEXPIRED);
-        if (doOffers || doLines || doNFTokenOffers)
+        bool const doCredentials = (result == tecEXPIRED);
```

Comment:
Why not just rename `doNFTokenOffers` to `doNFTokenOffersAndCredentials` to avoid having two booleans that are always going to have the exact same values? Or rename it to `doExpired` and leave a comment that says it's for `NFTokenOffers` and `Credentials`.

Reply:
Because credentials has nothing in common with tokens  and such a name will look weird. It will be easier to review and to change in the future if we will not merge different functionalities.